indico/modules/events/tracks/__init__.py

```diff
@@ -56,8 +54,30 @@ def _get_cloners(sender, **kwargs):
 
 
 @signals.acl.get_management_permissions.connect_via(Event)
-def _get_management_permissions(sender, **kwargs):
-    return TrackConvenerPermission
+def _get_event_management_permissions(sender, **kwargs):
+    yield TrackConvenerPermission
+    yield GlobalConvenePermission
+
+
+@signals.acl.get_management_permissions.connect_via(Track)
+def _get_track_management_permissions(sender, **kwargs):
+    yield ConvenePermission
+
+
+class ConvenePermission(ManagementPermission):
+    name = 'convene'
+    friendly_name = _('Convene')
+    description = _('Grants track convener rights in a track')
```

Comment:
Missing `.` or just redundant. (Since you have it in `GlobalConvenePermission.description`